ecosystem/sep-0024.md

```diff
@@ -689,7 +689,7 @@ Name | Type | Description
 `amount_fee` | string | Amount of fee charged by anchor.
 `amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information.
 `started_at` | UTC ISO 8601 string | Start date and time of transaction.
-`completed_at` | UTC ISO 8601 string | Completion date and time of transaction.  Assigned null for in-progress transactions.
+`completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
```

Comment:
What does this change mean in terms of the API response. Will optional mean that the response field will be omitted? There might have been a reason we made this explicitly null. I don't remember if this field specifically related to this, but the Vibrant team provided feedback at one point that having omitted fields is more difficult to debug and code against than null fields, and so we started preferring null fields.

Reply:
@leighmcculloch Yes, this means optional fields may be omitted. When we were looking at the example JSON API responses, optional fields seems to imply omission. 

@CassioMG Can you also provide feedback of making SEP24 consistent with SEP-6 and SEP-31 which mark `completed_at` as optional? Is Vibrant treating the `completed_at` field in SEP-24 different from SEP-6 and SEP-31?

Reply:
@lijamie98 it's completely fine to omit this field, we are not making any comparisons with `null`

Reply:
Also +1 for making it consistent throughout all SEPs, thanks for fixing this